src/main/java/org/generationcp/middleware/service/api/study/StudyService.java

```diff
@@ -77,4 +76,14 @@
 	Integer getPlotDatasetId(final int studyId);
 
 	Integer getEnvironmentDatasetId(final int studyId);
+
+	List<VariableDTO> getVariablesByDatasetId(final Integer pageSize, final Integer pageNumber,
```

Comment:
These services do not belong in `StudyService`

Reply:
@darla-leafnode , Maybe we can move this to DatasetService? How about the getAllVariables?